.kitchen.yml

```diff
@@ -17,15 +15,15 @@ platforms:
         - yum -y install initscripts 
       platform: rhel
       run_command: /usr/lib/systemd/systemd
-      privileged: true
       pid_one_command: /usr/lib/systemd/systemd
-  - name: ubuntu-16.04
```

Comment:
Any chance you re-add this platform? It's kind of to remove a platform, specially if it has some states/files in the formula specifically addressed to it.

Reply:
There is several opened issues related to Ubuntu 16.04 + Docker, may be that we could wait for Ubuntu 18.04 LTS to avoid bad and old behaviours? I could add commented entries to enable after the release of 18.04.

Reply:
We used [ubuntu 17.10](https://github.com/saltstack-formulas/packages-formula/blob/master/.kitchen.yml#L16-L20) here, maybe it can be added instead of 16.04? I know 16.04 is LTS, but better have it tested... 

Again, not blockers IMO, just things I'd like to keep, but can live without, atm :smile: 

@aboe76 wdyt? If you're OK with the current PR, we can merge.

Reply:
I don't mind so long that it is tested on a debian/ubuntu system.
I know that debian != ubuntu, I think to switch to 18.04 when it is ready is a good thing.